frontend/app/routes/$lang+/_public+/apply+/$id+/review-information.tsx

```diff
@@ -288,39 +296,43 @@ export default function ReviewInformation() {
         </dl>
         <h2 className="mt-8 text-2xl font-semibold ">{t('apply:review-information.dental-title')}</h2>
         <dl>
-          <DescriptionListItem term={t('apply:review-information.dental-private-title')}>
+          <DescriptionListItem term={t('apply:review-information.dental-insurance-title')}>
             {t('apply:review-information.dental-insurance-has-access')}
             {dentalInsurance === 'yes' ? t('apply:review-information.yes') : t('apply:review-information.no')}
             <p className="mt-4">
               <InlineLink id="change-access-dental" to="/">
-                {t('apply:review-information.dental-private-change')}
+                {t('apply:review-information.dental-insurance-change')}
               </InlineLink>
             </p>
           </DescriptionListItem>
-          <DescriptionListItem term={t('apply:review-information.dental-public-title')}>
+          <DescriptionListItem term={t('apply:review-information.dental-benefit-title')}>
             {dentalBenefit.federalBenefit.access === 'yes' ? (
               <div>
-                <span>{t('apply:review-information.dental-has-access')}</span>
+                <span>{t('apply:review-information.dental-benefit-has-access')}</span>
                 <ul className="ml-6 list-disc">
                   <li>{dentalBenefit.federalBenefit.benefit}</li>
                 </ul>
               </div>
             ) : (
-              t('apply:review-information.dental-federal-no-access')
+              t('apply:review-information.dental-benefit-federal-no-access')
             )}
             {dentalBenefit.provTerrBenefit.access === 'yes' ? (
               <div>
-                <span>{t('apply:review-information.dental-has-access')}</span>
+                <span>{t('apply:review-information.dental-benefit-has-access')}</span>
                 <ul className="ml-6 list-disc">
-                  <li>{dentalBenefit.provTerrBenefit.benefit}</li>
+                  <li>
+                    {i18n.language === 'fr'
+                      ? provincialTerritorialSocialPrograms.filter((program) => program.provinceTerritoryStateId === dentalBenefit.provTerrBenefit.province).find((x) => x.id === dentalBenefit.provTerrBenefit.benefit)?.nameFr
+                      : provincialTerritorialSocialPrograms.filter((program) => program.provinceTerritoryStateId === dentalBenefit.provTerrBenefit.province).find((x) => x.id === dentalBenefit.provTerrBenefit.benefit)?.nameEn}
```

Comment:
```suggestion
                    {provincialTerritorialSocialPrograms
                      .filter((p) => p.provinceTerritoryStateId === dentalBenefit.provTerrBenefit.province)
                      .find((p) => p.id === dentalBenefit.provTerrBenefit.benefit)?.[i18n.language === 'fr' ? 'nameFr' : 'nameEn']
                    }
```

Reply:
changed to use `getNameByLanguage` and naming